packages/quick_actions/quick_actions_android/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
+## 0.6.3
```

Comment:
Since this only affects tests, which don't impact clients of the plugin, you can remove the version bump from the pubspec and just use `NEXT` here.

`publishable` may still flag it because of `build.gradle`, but since it's a `testImplementation` line that's a false positive that you can override with the label. (Currently it's not doing any line-level analysis of the diff, only looking at files. Eventually we should make it smarter.)

Reply:
Oh I understand now. Ok thanks!